airbyte-integrations/connectors/destination-mysql/src/main/resources/spec.json

```
@@ -0,0 +1,49 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/mysql",
+  "supportsIncremental": true,
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "MySQL Destination Spec",
+    "type": "object",
+    "required": ["host", "port", "username", "database", "password"],
```

```suggestion
    "required": ["host", "port", "username", "database"],
```

I don't think password is actually required? You already have checks to see if it's set before using it elsewhere.

---

Jared, when is this true? When a DB doesn't set a password?

---

Yeah. We do this for other dbs as well. Mostly allows users to test if they have a local db they didn't add a password for.